backend/src/main/java/app/web/rest/GameRestController.java

```diff
@@ -1,57 +1,46 @@
 package app.web.rest;
 
 import app.domain.game.GameService;
-import app.domain.round.RoundService;
-import org.springframework.web.bind.annotation.*;
-
 import java.util.UUID;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
 
 @RestController
+@RequestMapping("/game")
 public class GameRestController {
 
     private final GameService gameService;
-    private final RoundService roundService;
 
-    public GameRestController(GameService gameService, RoundService roundService) {
+    public GameRestController(GameService gameService) {
         this.gameService = gameService;
-        this.roundService = roundService;
-    }
-
-    @PostMapping("/players/add")
-    public AddPlayerResponse addPlayer(@RequestParam String playerName) {
-        UUID playerId = gameService.joinToGame(playerName);
-        return new AddPlayerResponse(playerId.toString());
     }
 
-    @PostMapping("/game/start")
-    public void startRound() {
+    @PostMapping("/start")
+    public void startGame() {
         gameService.startGame();
     }
 
-    @PostMapping("/game/nextStage")
-    public void nextStage() {
-        roundService.startNextStage();
+    @PostMapping("/new-round")
+    public void newRound() {
+        gameService.startRound();
     }
 
-    @PutMapping("/game/blinds/update")
-    public void updateBlinds(@RequestParam int small) {
-        gameService.updateBlinds(small);
+    @PostMapping("/entry-fee")
+    public void setEntryFee(@RequestParam int entryFee) {
+        gameService.setEntryFee(entryFee);
     }
 
-    @PostMapping("/players/{id}/activation-status")
-    public void changePlayerActiveStatus(@PathVariable String id, @RequestParam boolean isActive) {
-        gameService.changeActiveStatus(UUID.fromString(id), isActive);
+    @PostMapping("/{id}/finish")
```

Comment:
może lepiej game/finish i jako requestParams id niż game/{id}/finish. Teraz ten id sugeruje że to jest id gry

Reply:
racja, i tak moim zdaniem bedzie potrzebny jakis dtos z tokenem, zrobie tak jak piszesz bedzie latwiej potem dodac ten token